crates/sel4-capdl-initializer/types/src/spec.rs

```diff
@@ -229,7 +229,7 @@ pub mod object {
     #[derive(Debug, Clone, Eq, PartialEq)]
     #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
     pub struct ArmIrqExtraInfo {
-        pub trigger: Word,
+        pub trigger: bool,
```

Comment:
I think it would be best to leave `trigger` as `Word` in CapDL-world. This would align with the rest of the CapDL code better. Plus, it wouldn't require any changes to the CapDL tool.

This will require a conversion in `capdl-initializer-core`.

Reply:
Ok, reverted that change and added a conversion in capdl-initializer-core, but not sure if that's the cleanest way to do the conversion or if I should implement TryInto<bool> for Word or something like that.